cupyx/scipy/interpolate/_ndgriddata.py

```diff
@@ -0,0 +1,161 @@
+"""
+Convenience interface to N-D interpolation
+"""
+
+import cupy
+from cupyx.scipy.interpolate._interpnd import (
+    NDInterpolatorBase, _ndim_coords_from_arrays)
+from cupyx.scipy.spatial import KDTree
+
+
+# ------------------------------------------------------------------------------
+# Nearest-neighbor interpolation
+# ------------------------------------------------------------------------------
+
+
+class NearestNDInterpolator(NDInterpolatorBase):
+    """NearestNDInterpolator(x, y).
+
+    Nearest-neighbor interpolator in N > 1 dimensions.
+
+    Methods
+    -------
+    __call__
```

Comment:
I think the `Methods` section needs to be removed to unblock the doc build. 
This is some incompatibility of the scipy and cupy doc builds : the former tolerates both autogenerated and manually added methods section, but the latter does not.